airbyte-integrations/connectors/source-retently/unit_tests/test_streams.py

```diff
@@ -18,10 +18,16 @@ def patch_base_class(mocker):
     mocker.patch.object(Companies, "__abstractmethods__", set())
 
 
-def test_request_params(patch_base_class):
+def test_request_params_companies(patch_base_class):
     stream = Companies()
     inputs = {"stream_slice": None, "stream_state": None, "next_page_token": None}
-    expected_params = None
+    expected_params = {'limit': 100}
+    assert stream.request_params(**inputs) == expected_params
+
+def test_request_params_other(patch_base_class):
```

Comment:
```suggestion

def test_request_params_other(patch_base_class):
```

Tests are not passing because of the blank line missed.